src/System.Private.ServiceModel/tools/setupfiles/BridgeCertificateBootstrapper.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+show_usage() 
+{
+    echo ""
+    echo "WCF Core Bridge CA Certificate Bootstrapper" 
+    echo "    Usage: $0 [bridge-host] [certificate-file]"
+    echo ""
+    echo "    bridge-host - hostname where Bridge is running" 
+    echo "    certificate-file - Full file path where the retrieved certificate should be saved"
+    echo ""
+}
+
+run_bootstrapper() 
+{
+    __bridge_host=$1
+    __outfile=$2
+
+     # Locate any CoreRun.exe to be able to execute BridgeCertificateBootstrapper.exe
```

Comment:
Nit: possibly echo __bridgeHost and __outFile.  Not required but I've found it useful for debugging.